Userland/Libraries/LibWeb/HTML/HTMLTableCellElement.cpp

```diff
@@ -61,6 +63,9 @@ void HTMLTableCellElement::apply_presentational_hints(CSS::StyleProperties& styl
             if (auto parsed_value = parse_nonzero_dimension_value(value))
                 style.set_property(CSS::PropertyID::Height, parsed_value.release_nonnull());
             return;
+        } else if (name == HTML::AttributeNames::background) {
+            style.set_property(CSS::PropertyID::BackgroundImage, CSS::ImageStyleValue::create(document().parse_url(value)).release_value_but_fixme_should_propagate_errors());
```

Comment:
We should check if the parsed url `is_valid()` before trying to make a StyleValue for it.